src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java

```diff
@@ -33,6 +34,8 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
         this.isModeratedQuestion = isModeratedQuestion;
         isRecipientNameHidden = questionAttributes.isRecipientNameHidden();
         isTeamQuestion = questionAttributes.giverType.equals(FeedbackParticipantType.TEAMS);
```

Comment:
Slightly out of the scope of this PR, but now that `isRecipientTeam` is added, this field should be renamed to `isGiverTeam` to reduce its ambiguity now.